### DISCUSSION TITLE
How to get more achiemevent?

### DISCUSSION THREAD

**Discussion by isro-ori** [community/community#65801] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I want a how to for more badges,
Thx in any advance.

**Comment by diegotorrent** [comment]
Take a look https://github.com/diegotorrent/Git-Badges

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
